src/backend/joanie/settings.py

```diff
@@ -298,6 +298,9 @@ class Base(Configuration):
     MARION_DOCUMENT_ISSUER_CHOICES_CLASS = "howard.defaults.DocumentIssuerChoices"
     MARION_CERTIFICATE_DOCUMENT_ISSUER = "howard.issuers.CertificateDocument"
 
+    # Contract Definition Issuer
+    JOANIE_CONTRACT_DEFINITION_ISSUER = "joanie.core.utils.issuers.generate_document"
```

Comment:
What we need in settings is a list of document names (that we use to build the path of the html and css file)